src/clj/rems/handler.clj

```diff
@@ -113,25 +113,24 @@
 (defn app-routes []
   (routes
    home-route
-   (middleware/wrap-no-cache
-    (wrap-login-redirect
-     (routes attachment-routes
-             redirects
-             ;; TODO /entitlements.csv should be an API
-             entitlements/entitlements-routes)))
+   (wrap-login-redirect
+    (routes attachment-routes
+            redirects
+            ;; TODO /entitlements.csv should be an API
+            entitlements/entitlements-routes))
+   (auth/auth-routes)
    styles/css-routes
-   ;; never cache authentication results
-   ;; TODO this is a slightly hacky place to do this
-   (middleware/wrap-no-cache (auth/auth-routes))
    #'api-routes
    ;; TODO should we disable logging of resource requests?
-   resource-handler
-   (extra-script-routes (:extra-scripts env))
-   (static-resources (:extra-static-resources env))
-   (static-resources (:theme-static-resources env))
-   webjar-handler
+   (wrap-cacheable
+    (routes
+     resource-handler
+     (extra-script-routes (:extra-scripts env))
+     (static-resources (:extra-static-resources env))
+     (static-resources (:theme-static-resources env))
```

Comment:
We could perhaps say something about caching in the docs and say how to best handle these resources updated by admins (i.e. that perhaps adding the new file under a new name is best)

Reply:
done